### PR TITLE
Add support for using AFT with BBB testing harness

### PR DESCRIPTION
**Pull request by skuusela**

### CODE REVIEW

File: cutters/gpiocutter.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        pass
+
+    def connect(self):
+        """
+        Turns power on
+
+        Returns:
+            None
+        """
+        os.system("echo 1 > /sys/class/gpio/gpio60/value")
```
Comment (igor-stoppa) [inline cutters/gpiocutter.py]: I think these values should be parameters.
Moving away from BBB is unlikely for now, but as we move on into emulating peripherals, we might find out that some other function we need is available through this pad and we might want to move it to a different location.

File: cutters/gpiocutter.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    def disconnect(self):
+        """
+        Turns power off
+
+        Returns:
+            None
+        """
+        os.system("echo 0 > /sys/class/gpio/gpio60/value")
```
Comment (igor-stoppa) [inline cutters/gpiocutter.py]: ditto

File: cutters/gpiocutter.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# coding=utf-8
+# Copyright (c) 2013-2016 Intel, Inc.
```
Comment (igor-stoppa) [inline cutters/gpiocutter.py]: The year of copyright should start from the time the code was written.

File: cutters/gpiocutter.py (hunk 1 above)
Comment (ghost) [reply cutters/gpiocutter.py]: I think we should remove the call to system besides we should add an Exception
This is my suggestion --> https://nopaste.linux-dev.org/?1120217
(It'll be online just for a week)

Comment (igor-stoppa) [reply cutters/gpiocutter.py]: We will probably need to use GPIOs also for emulating button presses (not necessarily with relays, for digital cases we should also support optical couplers), so I'd leave the name "relay" out from the base class. Another one can be derived for the specific relay/couple case - for example gpioswitch. The reason why I'd like to support both mechanical and optical switches is that they can be used to simulate different phenomenons. The mechanical switch will generate bounces, so it's good to test how the DUT can cope with it. The optical one, otoh, can be used to simulate other use cases (ex: end-of-line detection in a 3d printer or other similar contraption. Or optical encoder, to measure rotational speed).

But overall I agree that this should be improved.

However, if it works, I would rather merge this and then refine it in a follow up patch.

**Comment by skuusela** [comment]
Used your suggestion @eplauchu and changed it a bit. Now it's possible to change the GPIO pin used from configuration files and also set if 1 or 0 will put relay on or off.

**Comment by igor-stoppa** [comment]
@skuusela I do not see it as showstopper for merging this PR, however I would prefer to have generic handling of GPIOs - could that come in a following PR?

@eplauchu if you are happy with this, please give your +1, so that i can merge it

**Comment by skuusela** [comment]
@igor-stoppa Maybe not in the next PR because I think it's not that high in priority yet. I'll restructure this repo first and then remove the most important functionalities from the Beaglebone DAFT side and add them to server side DAFT.

**Pull request merged**